### PR TITLE
Add link to JA

### PR DESCRIPTION
**Pull request by logankilpatrick**
Add link to JA

### CODE REVIEW

**Comment by fredrikekre** [comment]
All the other links are to things run by the JuliaLang org so not sure this fits in there.

**Comment by logankilpatrick** [comment]
JuliaAcademy is run by the JuliaLang org ?

**Comment by ViralBShah** [comment]
I am guessing this readme is more for developers of Julia rather than users. JuliaAcademy is freely with MIT licensed materials (although the videos developed by Julia Computing continue to be copyrighted and owned by Julia Computing). I hope that we can recruit more videos onto JuliaAcademy - which naturally would be copyrighted by their creators.

Julia Computing does pay for the hosting and management of the Julia Academy platform on Teachable, but does not charge anything to the users and provides it as a community service to help grow the Julia user base.

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ helpful to start contributing to the Julia codebase.
 - [**StackOverflow**](https://stackoverflow.com/questions/tagged/julia-lang)
 - [**Twitter**](https://twitter.com/JuliaLanguage)
 - [**Meetup**](https://julia.meetup.com/)
+- [**Learn Julia on JuliaAcademy**](https://juliaacademy.com)
```
Comment (StefanKarpinski) [inline README.md]: Maybe this:
```suggestion
- [**Learning resources**](https://julialang.org/learning/)
```
Links to the learning page on JuliaLang.org which lists Julia Academy as the first resource, along with many others.

**Review by StefanKarpinski — APPROVED** [review]

**Pull request merged**